packages/moonstone/ExpandableInput/ExpandableInput.js

```diff
@@ -205,18 +205,6 @@ class ExpandableInputBase extends React.Component {
 		this.handleDeactivate = handleDeactivate.bind(this);
 	}
 
-	UNSAFE_componentWillReceiveProps (nextProps) {
-		let {initialValue} = this.state;
-		if (!this.props.open && nextProps.open) {
-			initialValue = nextProps.value;
-		} else if (this.props.open && !nextProps.open) {
-			this.paused.resume();
```

Comment:
@teckliew Did you verify ENYO-5734 doesn't reproduce on device after removing this?

Reply:
AFAIK `this.paused.resume()` is called here on `Deactivate`: https://github.com/enactjs/enact/pull/2062/files#diff-fb600147cf07922cd9d114f8156b234eL247
https://github.com/enactjs/enact/pull/2062/files#diff-fb600147cf07922cd9d114f8156b234eL28

Reply:
so it was actually redundant when `resume` was called on close.

Reply:
It would seem so but we added it specifically for that ticket. IIRC, `onDeactivate` wasn't fired because the expandable was closed before the input received focus so it couldn't fire the event.

Reply:
It doesn't reproduce, but I could add this back in at `ComponentDidMount`